content/articles/multi-output-classification-with-machine-learning/index.md

```diff
@@ -0,0 +1,303 @@
+<!--more-->
+
+An example of a multi-output classification model is a model that predicts the “type” and “color” of fruit at the same time. The “type of fruit” can be, “orange”, “mango” and “pineapple”. The “color” can be, “red”, “green”, “yellow”, and “orange". The multi-output classification is used to solve this problem and gives two prediction results.
+
+In this tutorial, we will build a multi-output text classification model using the Netflix dataset. The model will classify the input text as either `TV Show` or `Movie`. This will be the first output. The model will also classify the rating as: `TV-MA`, `TV-14`, `TV-PG`, `R`, `PG-13` and `TV-Y`.  The rating will be the second output. We will use Scikit-Learn `MultiOutputClassifier` algorithm to build this model.
+
+### Table of contents
+- [Prerequisites](#prerequisites)
+- [Netflix Dataset](#netflix-dataset)
+- [Loading the dataset](#loading-the-dataset)
+- [Output columns distribution](#output-columns-distribution)
+- [Text cleaning](#text-cleaning)
+- [Importing important packages](#importing-important-packages)
+- [Adding features and labels](#adding-features-and-labels)
+- [Dataset splitting](#dataset-splitting)
+- [Importing pipeline](#importing-pipeline)
+- [Making predictions](#making-predictions)
+- [Prediction probabilities](#prediction-probabilities)
+- [Conclusion](#conclusion)
+- [References](#references)
+
+### Prerequisites
+To understand the concepts in this tutorial, a reader should:
+
+- Understand [Python programming](/engineering-education/python-projects-for-beginners/)
+- Be able to build [machine learning models.](/engineering-education/house-price-prediction/)
+- Understand [natural langauge processing](/engineering-education/nlp-based-detection-model-using-neattext-and-scikit-learn/)
+- Know how to train models using [Scikit-learn](https://scikit-learn.org/stable/)
+- Use [Google Colab notebook](https://research.google.com/) to run the Python code in this tutorial.
+
+### Netflix dataset
+We will use the Netflix dataset to build our model. The image below will show how our dataset is structured.
+
+![Dataset image](/engineering-education/multi-output-classification-with-machine-learning/dataset-image.png)
+
+From the image above, our dataset has four columns: `title`, `description`, `type`, and `rating`. The `title` column will be the input column while type` and `rating` will be the output column. We now need to load this dataset on our machine.
+
+To download this Netflix dataset, click [here](https://drive.google.com/file/d/1524vMr5iCGEEJmPx2n2-TAl8EicKuxcg/view?usp=sharing)
+
+### Loading the dataset
+There are various exploratory data analysis (EDA) packages that will load our dataset. Let's import them using the following code:
+
+```python
+import pandas as pd
+import numpy as np
+```
+We will use Pandas to load the dataset. Numpy will be used to perform computational operations on our dataset. It also works well with arrays.
```

Comment:
Consider rewriting the sentence, "Numpy will be used..." in active voice. For instance, "We will use Numpy...."